ironmon_tracker/data/RouteData.lua

```diff
@@ -3308,7 +3322,13 @@ function RouteData.setupRouteInfoAsRSE()
 	}
 
 	RouteData.Info[54] = { name = "Mom's House", }
+	RouteData.Info[55] = { name = "Your Room", }
+	RouteData.Info[56] = { name = "Rival's House 1F", }
+	RouteData.Info[57] = { name = "Rival's House 2F", }
```

Comment:
Disagree with these, double-checked and my initial thought was right
Brendan's house is the left house and May's house is the right house, these are hardcoded therefore if you play as Brendan (male character) then these line up fine however if you instead choose May (female character) they don't. See below screenshots

![Screenshot_20](https://user-images.githubusercontent.com/106463662/208412295-19a6bdc6-5e86-4fcd-91f7-f8a500dc9088.png)
![Screenshot_21](https://user-images.githubusercontent.com/106463662/208412301-da17d130-71b8-4506-a76b-b0fe8d14a1a7.png)
![Screenshot_22](https://user-images.githubusercontent.com/106463662/208412303-49bb7e8a-7f1b-41f5-8f01-bb47dbb4ca35.png)

Not that it's a huge issue ultimately (bizhawk users won't see it as there's no mons there to encounter and no current way to lookup routes outside of encounters) but imo should probably be "Brendan's House/Room" and "May's House/Room" (or 1F/2F equivalents)

Reply:
Updated using solution of:
- "Brendan's House 1F"
- "Brendan's House 2F"
- "May's House 1F"
- "May's House 2F"